src/main/java/StringAddCalculator.java

```diff
@@ -0,0 +1,32 @@
+public class StringAddCalculator {
+
+    public static final String CUSTOM_DELIMITER_PATTERN = "//(.)\n(.*)";
+    public static final String COMMA = ",";
+    public static final String COLON = ":";
+    public static final int NUMBER_ZERO = 0;
+
+    public static int splitAndSum(String inputText) {
+        if (inputText == null || inputText.isEmpty()) return NUMBER_ZERO;
```

Comment:
```suggestion
        if (inputText == null || inputText.isEmpty()) {
                return NUMBER_ZERO;
        }
```

한라인도 괄호로 감싸주는 것이 컨벤션입니다 :)